.github/ISSUE_TEMPLATE/i-have-a-problem-with-rive-flutter.md

```diff
@@ -0,0 +1,59 @@
+---
+name: I have a problem with rive-flutter
+about: Tell us what's wrong
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<-- 
+Thanks for using Rive!
+
+If something isn't working like you expected, please:
+- Make sure you've updated to rive-flutter to the latest version:
+			https://pub.dev/packages/rive/versions
+- Check out our help center
+			https://help.rive.app/
+
+   If you found a bug or if you can't find what you're looking for 
+in our docs, fill out the template below.
+-->
+
+### Description
+<-- A clear and concise description of what the problem is. -->
+
+### Steps To Reproduce
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error ⚠️
+
+#### Source `.riv`/`.rev` file
+<-- 
+	Attaching the source `.riv` file that reproduces the issue is immensely helpful for our team to identify the problem as fast as possible and fix it.
+	If you can also attach the source (`.rev`) by [downloading a backup](https://help.rive.app/editor/exporting#for-backup) from the Editor, that will help us speed things up too.
```

Comment:
Maybe just slim this down to, "Please also include the source (`.rev`)..."

Reply:
I'll add a note at the end of that paragraph, suggesting to email either file to us if they don't want to share it publicly

Reply:
Might want to also note that they should zip them up together as GitHub doesn't accept .riv or .rev (or email them to us if IP is an issue).  What email address should we suggest for that? info@rive.app?

Reply:
I added hello@rive.app, the preview hasn't updated yet!